docs/plugins/testing_workshop_docs/index.md

```diff
@@ -0,0 +1,5 @@
+This is the frontpage for the testing workshop articles. 
```

Comment:
This document needs a title so that is it rendered in the ToC correctly. Maybe

```suggestion
# Testing workshop

This is the frontpage for the testing workshop articles. 
```

Reply:
Yes.  Will do.

Reply:
Done.